File: ezcode/레벨2/68. 특정 달의 일수 구하기/특정 달의 일수 구하기.py
year, month = map(int, input().split())

# 각 달의 일 수 (인덱스 0은 사용하지 않음)
days = [0, 31, 28, 31, 30, 31, 30,
        31, 31, 30, 31, 30, 31]

# 윤년이면 2월을 29일로 설정
if (year % 4 == 0 and year % 100 != 0) or (year % 400 == 0):
    days[2] = 29

print(days[month])

File: ezcode/레벨2/68. 특정 달의 일수 구하기/특정 달의 일수 구하기.c
#include <stdio.h>

int main() {
    int y, m;
    scanf("%d %d", &y, &m); // y: year, m: month

    int days[] = { 0, 31, 28, 31, 30, 31, 30,
                   31, 31, 30, 31, 30, 31 };

    // 윤년이면 2월을 29일로
    if ((y % 4 == 0 && y % 100 != 0) || (y % 400 == 0)) {
        days[2] = 29;
    }

    printf("%d\n", days[m]);

    return 0;
}
